src/numerical_flux/entropy_cons_numerical_flux.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef __ENTROPY_CONS_NUM_FLUX__
+#define __ENTROPY_CONS_NUM_FLUX__
+
```

Comment:
indent entire contents of file accordingly

Reply:
fixed